.travis.yml

```diff
@@ -54,3 +56,5 @@ script:
   - cd docs && make html && cd ..
   # Make sure the library installs.
   - python setup.py install
+    # Make sure benchmarks runs
+  - benchmark_pydy_code_gen.py 2 4
```

Comment:
Also note that matlplotlib is not installed on Travis. Either we need to install it or make the plot optional in the benchmark script. I think I'd prefer the latter.